pkg/models/licenses.go

```diff
@@ -0,0 +1,3 @@
+package models
+
+type Licenses []string
```

Comment:
I've noticed that we don't really do type aliases in osv-scanner, maybe to safeguard against us ever wanting to add to this Licenses field in the future?

Maybe this should be

```
type Licenses struct {
   Licenses []string
}
```

Also this should probably go in the results.go file.

Reply:
I think a
```
type License string
```
could work better, and `Licenses` field will be `[]License`.

Reply:
changed this to a type License string.